src/transports/ws.rs

```diff
@@ -512,13 +548,16 @@ pub mod compat {
     /// TLS stream type for tokio runtime.
     #[cfg(feature = "ws-tls-tokio")]
     pub type TlsStream = Compat<async_native_tls::TlsStream<tokio::net::TcpStream>>;
+    /// Rustls TLS stream type for tokio runtime.
+    #[cfg(all(feature = "ws-rustls-tokio", not(feature = "ws-tls-tokio")))]
```

Comment:
```suggestion
    #[cfg(feature = "ws-rustls-tokio")]
```

After we explicitly panic during build time when both `ws-tls-tokio` and `ws-rustls-tokio` are enabled.